src/main/java/nextstep/courses/domain/SessionRegisterInfo.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.courses.domain;
+
+import nextstep.users.domain.NsUser;
+
+public class SessionRegisterInfo {
+    private final SessionStatus sessionStatus;
+    private final Students students = new Students();
+    private final Payments payments = new Payments();
+    public SessionRegisterInfo(SessionStatus sessionStatus) {
```

Comment:
인스턴스 변수와 생성자의 구분을 위해 공백 라인을 사이에 추가해보면 좋을 것 같습니다.
```suggestion
    private final Payments payments = new Payments();
    
    public SessionRegisterInfo(SessionStatus sessionStatus) {
        this.sessionStatus = sessionStatus;
    }
```